src/data/data_fetcher.py

```diff
@@ -231,8 +234,8 @@ def get_next_page(self):
         except:
             url = self.starting_url
 
-        uh = urllib.request.urlopen(url)
-        data = uh.read().decode()
+        uh = requests.get(url)
+        data = uh.text
         self.info = json.loads(data)
 
     def get_transactions_on_page(self):
```

Comment:
While we're in the nuts and bolts of the `DataFetcher` class, let's make these functions, and their references, `_hidden`